src/modules/back-to-top/back-to-top.component.scss

```diff
@@ -0,0 +1,33 @@
+@import '../../styles/_variables';
+
+.back-to-top {
+  position: fixed;
+  bottom: 10px;
+  right: 10px;
+  color: $sky-color-white;
```

Comment:
Sorry, this is my fault for not explaining correctly: we should always use "stache" variables in our "stache" stylesheets. These variables can certainly reference "sky" variables in the `_variables.scss` file, but it helps isolate features between the two libraries (see `$stache-color-blue` as an example). 

For example, what if SKY UX decided to change the variable `$sky-color-white` to `$skyux-white`? We'd have to do a find and replace in all of our stylesheets to make the change.